coding-concepts.js

```diff
@@ -37,23 +38,24 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: index elements start at zero - first element counted as zero, then one, two, ++. In this array, "Ruby" is the second element (first index)
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: SATURDAY, SUNDAY
+// b) Verify and explain: returned not a function - error - need to loop through array to add uppercase to each string in array
 
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean"]
-// console.log(typeof dataTypes.length)
+console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: number (initial thought is length of string)
+// b) Verify and explain: Output is number - function executed the typeOf command on dataTypes. Number is the first data type.
+//.length was not executed because iteration was not assigned?
```

Comment:
dataTypes.length will produce 3 and typeof will evaluate that output as a number.